lib/archethic/p2p/message/get_current_replication_attestations.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Archethic.P2P.Message.GetCurrentReplicationAttestations do
+  @moduledoc """
+  A request to get the replication attestations of current summary
+  The nodes receiving this must be elected to store the given subsets
+  """
+
+  @enforce_keys [:subsets]
+  defstruct [:subsets]
+
+  alias Archethic.BeaconChain.ReplicationAttestation
+  alias Archethic.Crypto
+  alias Archethic.BeaconChain
+  alias Archethic.P2P.Message.GetCurrentReplicationAttestationsResponse
+
+  @type t :: %__MODULE__{subsets: list(binary())}
+
+  @spec process(message :: __MODULE__.t(), sender_public_key :: Crypto.key()) ::
+          GetCurrentReplicationAttestationsResponse.t()
+  def process(%__MODULE__{subsets: subsets}, _) do
+    %GetCurrentReplicationAttestationsResponse{
+      replication_attestations:
+        subsets
+        |> Stream.flat_map(&BeaconChain.get_current_summary_replication_attestations/1)
+        |> ReplicationAttestation.reduce_confirmations()
```

Comment:
Since the reducing confirmations is already done in the BeaconChain function, it is not needed to do it here as it isn't possible to have attestation for the same transaction on different subset